### PR TITLE
Add a background and border-bottom to featured images.

### LINKED ISSUE #15: Featured image sizes / size limits
The design calls for full-bleed featured images. This poses some interesting challenges:
1. Should the images have a maximum display height? If so, how is increasing width handled in terms of cropping?
2. What happens when the author uploads an image that is narrower than the current viewport width?
3. What happens when the site is displayed on a very wide or zoomed out screen?

Some of this is squarely a design decision, some of it has to do with implementation.

### PR DESCRIPTION
**Pull request by melchoyce**
Fixes #15.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1511,6 +1511,11 @@ body:not(.twentyseventeen-front-page) .entry-header {
 	max-width: 740px;
 }
 
+.single-featured-image-header {
+    background-color: #fafafa;
```
Comment (grappler) [inline style.css]: This should be a tab and not spaces

**Review by grappler — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (melchoyce) [reply style.css]: The rest of the stylesheet seems to be spaces, though?

Comment (grappler) [reply style.css]: I see something different

image

Comment (melchoyce) [reply style.css]: Weird... I'll copy/paste from existing indents and see if that fixes it.

**Review by grappler — APPROVED** [review]

**Comment by melchoyce** [comment]
Thanks @grappler.

**Pull request merged**